tests/unit/services/embedded-test.ts

```diff
@@ -1,5 +1,6 @@
 import { module, test } from 'qunit'
 import { setupTest } from 'ember-qunit'
+import EmbeddedService from 'ember-cli-embedded/services/embedded'
```

Comment:
Since we only import this symbol as a type and not a value.

```suggestion

import type EmbeddedService from 'ember-cli-embedded/services/embedded'
```